packages/blazor-workspace/NimbleBlazor/package.json

```diff
@@ -26,6 +26,6 @@
   ],
   "peerDependencies": {
     "cross-env": "^7.0.3",
-    "rimraf": "^5.0.5"
```

Comment:
The PR description says rollup should update but the resolved rollup didn't actually change. 

Turns out the rollup-plugin-sourcemaps
 package we depend on hasn't been updates in years and has a peerdep on rollup 2 so we are stuck on that for the root package resolution. Looks like it's unmaintained: https://github.com/maxdavidson/rollup-plugin-sourcemaps/issues/130

And should maybe switch to https://github.com/2wce/rollup-plugin-sourcemaps2

Reply:
Actually this package doesn't seem to be used.. https://www.npmjs.com/package/rollup-plugin-sourcemaps2
This isn't causing an actual issue as far as I can tell so just gonna ignore for now

Reply:
It also failed the build because the package is missing files the script tried to import.